settings.gradle

```diff
@@ -15,3 +15,5 @@
 
 include 'core', 'examples', 'clients', 'tools', 'streams', 'streams:examples', 'log4j-appender',
         'connect:api', 'connect:transforms', 'connect:runtime', 'connect:json', 'connect:file', 'jmh-benchmarks'
+include 'log4j-controller'
```

Comment:
Also, a question: is it fine to do this whole module separation thing? I'm a little bit concerned about the implications but I don't think there is a good way to solve it (as in fully backward compatible and transparent for the users).
I think that in the long term Kafka should recommend logback as it is more robust and offers much more management features (including the ones that Log4jController provides) but it is a necessary step to achieve this to separate log4j stuff off kafka-core.
Do you have any suggestions about how do we present it to users? I'm basically thinking about something like the log4j appender: users can grab the jar, put it to the classpath and the everything works as it did before.